src/Language/PureScript/Docs/Types.hs

```diff
@@ -48,7 +63,6 @@ data Package a = Package
   , pkgTagTime              :: Maybe UTCTime
   , pkgModules              :: [Module]
   , pkgModuleMap            :: Map P.ModuleName PackageName
-  , pkgResolvedDependencies :: [(PackageName, Version)]
```

Comment:
How would we get the specific versions without running the `bower` executable though?